pjsip/src/pjsua-lib/pjsua_vid.c

```diff
@@ -994,6 +994,8 @@ static pj_status_t setup_vid_capture(pjsua_call_media *call_med)
     /* Done */
     inc_vid_win(wid);
     call_med->strm.v.cap_win_id = wid;
+    PJ_LOG(4,(THIS_FILE, "Call %d: video capture set up with dev %d, wid=%d",
+	      call_med->call->index, call_med->strm.v.cap_dev, wid));
```

Comment:
better put the initial patch in a separate commit

Reply:
#2303

Reply:
Meant to be separate commits in the same PR (not sure how to do that, need to find out later perhaps). No probs with a new PR anyway.